src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.7.1
+
+Released 2024-Feb-09
+
```

Comment:
This has no mention of #5252

Reply:
Unless I'm mistaken, there's really nothing of note in #5252 that is user facing.

Reply:
In the CHANGELOGs, we have previously mentioned stuff that is not user facing but is a significant enough change. From this PR description, it looks like #5252 is the major driver for the new release. I think that it would be useful information for us or the instrumentation folks to look at in the CHANGELOG.